### PR TITLE
[bn] Localize(updated) content/bn/releases/_index.md

### PR DESCRIPTION
**Pull request by saifulshark**
Localize _index.md in Bengali language.

fixes issue:#44993

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@saifulshark Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

File: content/bn/releases/_index.md [hunk 1]
```diff
@@ -2,19 +2,24 @@
 linktitle: প্রকাশের ইতিহাস
 title: প্রকাশনাসমূহ
 type: docs
+layout: প্রকাশ-তথ্য
+notoc: সত্য
```
Comment (asem-hamid) [inline content/bn/releases/_index.md:6]: you don't have to localize them

File: content/bn/releases/_index.md [hunk 2]
```diff
@@ -2,19 +2,24 @@
...
 ---
 
-
 <!-- overview -->
 
-কুবারনেটিস প্রজেক্ট সাম্প্রতিক তিনটি পর্যন্ত ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}) ছোট রিলিজের জন্য রিলিজ শাখা বজায় রাখে।  কুবারনেটিস 1.19 এবং নতুন ভার্সন আনুমানিক 1 বছরের প্যাচ সমর্থন পায়। কুবারনেটিস 1.18 এবং তার বেশি বয়সীরা প্রায় 9 মাস প্যাচ সমর্থন পেয়েছে। 
+কুবারনেটিস প্রজেক্ট সাম্প্রতিক তিনটি পর্যন্ত ছোট রিলিজের জন্য রিলিজ শাখা বজায় রাখে।
+({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}})
+কুবারনেটিস 1.19 এবং নতুন ভার্সন 
+[আনুমানিক 1 বছরের প্যাচ সাপোর্ট পায়(patch support)](/releases/patch-releases/#support-period)
```
Comment (asem-hamid) [inline content/bn/releases/_index.md:14]: (/bn/releases/patch-releases/#support-period)

File: content/bn/releases/_index.md [hunk 3]
```diff
@@ -2,19 +2,24 @@
...
+কুবারনেটিস 1.18 এবং তার বেশি বয়সীরা প্রায় 9 মাস প্যাচ সাপোর্ট (patch support) পেয়েছে। 
 
 কুবারনেটিস সংস্করণ **x.y.z** হিসাবে প্রকাশ করা হয়,
-যেখানে **x** হল প্রধান সংস্করণ, **y** হল অপ্রধান সংস্করণ এবং **z** হল প্যাচ সংস্করণ, যা [শব্দার্থিক সংস্করণ](https://semver.org/) পরিভাষা অনুসরণ করে হয়। 
+যেখানে **x** হল মুখ্য সংস্করণ, **y** হল গৌণ সংস্করণ এবং **z** হল প্যাচ ভার্সন (patch version),
+যা [শব্দার্থিক সংস্করণ](https://semver.org/) পরিভাষা অনুসরণ করে হয়। 
 
 অতিরিক্ত তথ্যসমূহ [version skew policy](/releases/version-skew-policy/) নথিতে সংরক্ষিত রয়েছে।
```
Comment (asem-hamid) [inline content/bn/releases/_index.md:21]: (/bn/releases/version-skew-policy/)

File: content/bn/releases/_index.md [hunk 4]
```diff
@@ -23,6 +28,6 @@ type: docs
 
 ## আসন্ন রিলিজ
 
-কুবারনেটিস মুক্তিপ্রকাশ এর আসন্ন **{{< skew nextMinorVersion >}}** [সময়সূচী](https://github.com/kubernetes/sig-release/tree/master/releases/release-{{< skew nextMinorVersion >}}) চেক করুন!
+কুবারনেটিস রিলিজ এর আসন্ন **{{< skew nextMinorVersion >}}** [সময়সূচী](https://github.com/kubernetes/sig-release/tree/master/releases/release-{{< skew nextMinorVersion >}}) চেক করুন!
```
Comment (asem-hamid) [inline content/bn/releases/_index.md:31]: break this line into 2 line as en

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by saifulshark** [comment]
Successfully signed the agreement as an individual.
@asem-hamid did the work based on your review.
Thanks.

File: content/bn/releases/_index.md [hunk 5]
```diff
@@ -2,18 +2,23 @@
...
 ---
 
-
 <!-- overview -->
 
-কুবারনেটিস প্রজেক্ট সাম্প্রতিক তিনটি পর্যন্ত ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}) ছোট রিলিজের জন্য রিলিজ শাখা বজায় রাখে।  কুবারনেটিস 1.19 এবং নতুন ভার্সন আনুমানিক 1 বছরের প্যাচ সমর্থন পায়। কুবারনেটিস 1.18 এবং তার বেশি বয়সীরা প্রায় 9 মাস প্যাচ সমর্থন পেয়েছে। 
+কুবারনেটিস প্রজেক্ট সাম্প্রতিক তিনটি পর্যন্ত ছোট রিলিজের জন্য রিলিজ শাখা বজায় রাখে।
+({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}).
+কুবারনেটিস 1.19 এবং নতুন ভার্সন 
+[আনুমানিক 1 বছরের প্যাচ সাপোর্ট পায়(patch support)](bn/releases/patch-releases/#support-period)
```
Comment (asem-hamid) [inline content/bn/releases/_index.md:14]: /bn/releases/patch-releases/#support-period

File: content/bn/releases/_index.md [hunk 6]
```diff
@@ -2,18 +2,23 @@
...
+কুবারনেটিস 1.18 এবং তার বেশি বয়সীরা প্রায় 9 মাস প্যাচ সাপোর্ট (patch support) পেয়েছে। 
 
 কুবারনেটিস সংস্করণ **x.y.z** হিসাবে প্রকাশ করা হয়,
-যেখানে **x** হল প্রধান সংস্করণ, **y** হল অপ্রধান সংস্করণ এবং **z** হল প্যাচ সংস্করণ, যা [শব্দার্থিক সংস্করণ](https://semver.org/) পরিভাষা অনুসরণ করে হয়। 
-
-অতিরিক্ত তথ্যসমূহ [version skew policy](/releases/version-skew-policy/) নথিতে সংরক্ষিত রয়েছে। 
+যেখানে **x** হল মুখ্য সংস্করণ, **y** হল গৌণ সংস্করণ এবং **z** হল প্যাচ ভার্সন (patch version),
+যা [শব্দার্থিক সংস্করণ](https://semver.org/) পরিভাষা অনুসরণ করে হয়। 
 
+অতিরিক্ত তথ্যসমূহ [version skew policy](bn/releases/version-skew-policy/) নথিতে সংরক্ষিত রয়েছে।
```
Comment (asem-hamid) [inline content/bn/releases/_index.md:21]: /bn/releases/version-skew-policy/

Comment (saifulshark) [reply content/bn/releases/_index.md]: did this.

**Comment by saifulshark** [comment]
@asem-hamid please check out now,did this.

**Review by asem-hamid — APPROVED** [review]

**Pull request merged**